pkg/plugins/backendplugin/backend_plugin.go

```diff
@@ -185,14 +185,27 @@ func (p *BackendPlugin) CollectMetrics(ctx context.Context, ch chan<- prometheus
 	return nil
 }
 
-func (p *BackendPlugin) checkHealth(ctx context.Context) (*pluginv2.CheckHealthResponse, error) {
+func (p *BackendPlugin) checkHealth(ctx context.Context, config *PluginConfig) (*pluginv2.CheckHealthResponse, error) {
 	if p.diagnostics == nil || p.client == nil || p.client.Exited() {
 		return &pluginv2.CheckHealthResponse{
 			Status: pluginv2.CheckHealthResponse_UNKNOWN,
 		}, nil
 	}
 
-	res, err := p.diagnostics.CheckHealth(ctx, &pluginv2.CheckHealthRequest{})
+	jsonDataBytes, err := config.JSONData.ToDB()
+	if err != nil {
+		return nil, err
+	}
+
+	pconfig := &pluginv2.PluginConfig{
+		OrgId:                   config.OrgID,
+		PluginId:                config.PluginID,
+		JsonData:                jsonDataBytes,
+		DecryptedSecureJsonData: config.DecryptedSecureJSONData,
+		LastUpdatedMS:           config.Updated.UnixNano() / int64(time.Millisecond),
```

Comment:
Isn't it clearer to use 1000 instead of `time.Millisecond` since it's obvious that you divide a nanosecond by 1000 to get the corresponding millisecond value?

Reply:
Using the predefined consts in `time` is quite common. 

This makes it easy to convert nanoseconds to any time unit without knowning many zeros to devide by. Ex
```go
x.UnixNano() / int64(time.Second)
x.UnixNano() / int64(time.Hour)
```

Reply:
@bergquist Sorry I meant to cancel that comment, after thinking and reading the time package :)